src/ExpressionEvaluator/CSharp/Test/ResultProvider/TupleTests.cs

```diff
@@ -819,6 +819,123 @@ public void TupleElementNames_IncorrectCount()
             }
         }
 
+        // Different number of tuple elements
```

Comment:
Yes, I wrote these tests in a separate branch for #13420, but that bug may not be fixed soon so I merged the tests here so the tests are preserved in the meantime.